eng/common/scripts/Verify-RequiredDocsJsonMembers.ps1

```diff
@@ -91,6 +91,13 @@ function Test-RequiredDocsJsonMembers($moniker) {
       $script:FoundError = $true
     }
 
+    if ($fileObject.PSObject.Members.Name -contains 'DirectoryPath') {
```

Comment:
Should we treat the absence of the DirectoryPath property as a failure as well?

Reply:
This is done